pkg/util/intern/string.go

```diff
@@ -102,6 +103,7 @@ func (s *StringInterner) GetString(k string) *StringValue {
 // The returned pointer will be the same for Get(v) and Get(v2)
 // if and only if v == v2. The returned pointer will also be the same
 // for a string with same contents as the byte slice.
+//
```

Comment:
This formatting change should probably not be in this PR since it has nothing to do with this file.